rules/linux/lnx_local_account.yml

```diff
@@ -0,0 +1,22 @@
+title: Local System Accounts Discovery
+id: b45e3d6f-42c6-47d8-a478-df6bd6cf534c
+status: experimental
+description: Detects enumeration of local systeam accounts
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/08
+references:
+  - https://attack.mitre.org/techniques/T1087/001/
```

Comment:
Hello @alejandroortuno! Could you please remove reference to the ATT&CK as it already mentioned in the `tags` filed?